doodle-formats/src/format/deflate.rs

```diff
@@ -83,7 +83,7 @@ fn dist8(base: &BaseModule) -> Format {
 /// standard LSB-to-MSB write order for numeric values in the DEFLATE specification.
 fn bits8(n: usize, base: &BaseModule) -> Format {
     if n == 0 {
-        return Format::Compute(Expr::U8(0));
+        return Format::Compute(Box::new(Expr::U8(0)));
```

Comment:
I guess a helper `compute` function might not hurt.

Reply:
Yeah, I thought about this too as I was changing all of the unboxed Expr to be boxed...will author a new commit that hopefully simplifies such common-cases.